### PR TITLE
Added quiality-of-life improvements

### PR DESCRIPTION
**Pull request by CodeLongAndProsper90**
Added check for screen clear for non-UNIX systems, exit codes for various errors, and uses UNIX time instead of datetime.

### CODE REVIEW

**Review by ayushkumar0208 — APPROVED** [review]

File: PORT SCANNER.PY [hunk 1]
```diff
@@ -71,10 +71,11 @@ Open up an text editor, copy & paste the code below. Save the file as:
 import socket
```
Comment (bupboi1337) [inline PORT SCANNER.PY:71]: Make sure to list the requirements in a txt file.

File: PORT SCANNER.PY [hunk 2]
```diff
@@ -102,21 +103,21 @@ try:
...
     print("Couldn't connect to server")
-    sys.exit()
+    sys.exit(3)
 
 # Checking the time again
-t2 = datetime.now()
+t2 = time()
 
 # Calculates the difference of time, to see how long it took to run the script
 total = t2 - t1
```
Comment (bupboi1337) [inline PORT SCANNER.PY:119]: Replace total with runtime for extra readability or t3 for smaller file size

File: PORT SCANNER.PY [hunk 3]
```diff
@@ -71,10 +71,11 @@ Open up an text editor, copy & paste the code below. Save the file as:
 import socket
 import subprocess
 import sys
-from datetime import datetime
+from time import time
```
Comment (bupboi1337) [inline PORT SCANNER.PY:74]: Why time instead of datetime?

**Review by bupboi1337 — APPROVED** [review]
Looks great to me!

**Pull request merged**